third_party/kiva-ep/src/libkiva/Ground.hpp

```diff
@@ -22,6 +22,11 @@
 #pragma warning(push)
 #pragma warning(disable : 4127) // Conditional expression is constant
 #pragma warning(disable : 4459) // Declaration hides global declaration
+#elif defined(__clang__)
+#pragma clang diagnostic push
+#if __clang_major__ >= 13
+#pragma clang diagnostic ignored "-Wdeprecated-copy"
+#endif
```

Comment:
No problem here.  As long as it compiles and runs OK then how you handle things inside Kiva is not too much of my concern.  Thanks